coderd/database/db2sdk/db2sdk.go

```diff
@@ -659,3 +659,17 @@ func Organization(organization database.Organization) codersdk.Organization {
 		IsDefault:   organization.IsDefault,
 	}
 }
+
+func CryptoKeys(keys []database.CryptoKey) []codersdk.CryptoKey {
+	return List(keys, CryptoKey)
+}
+
+func CryptoKey(key database.CryptoKey) codersdk.CryptoKey {
+	return codersdk.CryptoKey{
+		Feature:   codersdk.CryptoKeyFeature(key.Feature),
+		Sequence:  key.Sequence,
+		StartsAt:  key.StartsAt.UTC(),
```

Comment:
The main motivatino here is `testify` which is being rather obnoxious about trying to do `Equal` comparisons between objects because there's a TZ discrepancy. E.g. 

```bash
        	Error:      	[]codersdk.CryptoKey{codersdk.CryptoKey{Feature:"workspace_apps", Secret:"462543a101f54c2aa51cbeeede315ef2d5a1bb6b15eb7f3df6c0a948e6ce64999650d7effc3da276c84d0e5ae34122f4225c2f12df75cacd8752b06faebc414b6ea98e1dba072988836b47fc8f711788c26feed32ed3a4a578c3467b8bfeea7c", DeletesAt:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Sequence:3, StartsAt:time.Date(2024, time.September, 29, 23, 55, 41, 913601000, time.Location(""))}, codersdk.CryptoKey{Feature:"workspace_apps", Secret:"a2f6d56687565f9407ab5710baabb0db82732567e0abbee17846cfd95868a3f0625e7e3ace10e29933d463c3d4068b806205c132a49fdac14fd336db20c0230157ac60bfb8e8067f18e41bddda1f7498374342a9088a33139aa9ac35409d5e8c", DeletesAt:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Sequence:2, StartsAt:time.Date(2024, time.September, 29, 22, 55, 41, 913601000, time.Location(""))}} does not contain codersdk.CryptoKey{Feature:"workspace_apps", Secret:"a2f6d56687565f9407ab5710baabb0db82732567e0abbee17846cfd95868a3f0625e7e3ace10e29933d463c3d4068b806205c132a49fdac14fd336db20c0230157ac60bfb8e8067f18e41bddda1f7498374342a9088a33139aa9ac35409d5e8c", DeletesAt:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Sequence:2, StartsAt:time.Date(2024, time.September, 29, 22, 55, 41, 913601000, time.Location("Canada/Newfoundland"))}
```

Reply:
Fine in test code, I guess, but I don't like seeing all the unnecessary `UTC()` calls in the product code.